lib/models/application.coffee

```diff
@@ -694,6 +694,180 @@ getApplicationModel = (deps, opts) ->
 			throw err
 		.asCallback(callback)
 
+	###*
+	# @summary Get whether the application is configured to track the latest release
+	# @name isTrackingLatestRelease
+	# @public
+	# @function
+	# @memberof resin.models.application
+	#
+	# @param {String|Number} nameOrId - application name (string) or id (number)
+	# @fulfil {Boolean} - is tracking the latest release
+	# @returns {Promise}
+	#
+	# @example
+	# resin.models.application.isTrackingLatestRelease('MyApp').then(function(isEnabled) {
+	# 	console.log(isEnabled);
+	# });
+	#
+	# @example
+	# resin.models.application.isTrackingLatestRelease(123).then(function(isEnabled) {
+	# 	console.log(isEnabled);
+	# });
+	#
+	# @example
+	# resin.models.application.isTrackingLatestRelease('MyApp', function(error, isEnabled) {
+	# 	console.log(isEnabled);
+	# });
+	###
+	exports.isTrackingLatestRelease = (nameOrId, callback) ->
+		exports.get(nameOrId, $select: 'should_track_latest_release')
+		.get('should_track_latest_release')
+		.asCallback(callback)
+
+	###*
+	# @summary Set a specific application to run a particular release
+	# @name setToRelease
+	# @public
+	# @function
+	# @memberof resin.models.application
+	#
+	# @description Configures the application to run a particular release
+	# and not get updated when the latest release changes.
+	#
+	# @param {String|Number} nameOrId - application name (string) or id (number)
+	# @param {String} fullReleaseHash - the hash of a successful release (string) or id (number)
+	# @returns {Promise}
+	#
+	# @example
+	# resin.models.application.setToRelease('MyApp', 'f7caf4ff80114deeaefb7ab4447ad9c661c50847').then(function() {
+	# 	...
+	# });
+	#
+	# @example
+	# resin.models.application.setToRelease(123, 'f7caf4ff80114deeaefb7ab4447ad9c661c50847').then(function() {
+	# 	...
+	# });
+	#
+	# @example
+	# resin.models.application.setToRelease('MyApp', 'f7caf4ff80114deeaefb7ab4447ad9c661c50847', function(error) {
+	# 	if (error) throw error;
+	# 	...
+	# });
+	###
+	exports.setToRelease = (nameOrId, fullReleaseHash, callback) ->
```

Comment:
Nit: How about `pinToRelease`? (just spotted - you actually suggested this! Good suggestion :+1:)

Reply:
Also changed that in the device model